StRoot/StSpinPool/StFcsPi0FinderForEcal/StFcsPi0FinderForEcal.cxx

```diff
@@ -228,15 +228,18 @@ Int_t StFcsPi0FinderForEcal::Make() {
       mNEvents++;
       cout << "current event:" << mNEvents << endl;
       if (mFilter == 1 && mFcsColl->numberOfHits(0) + mFcsColl->numberOfHits(1) + mFcsColl->numberOfHits(2) + mFcsColl->numberOfHits(3) == 0) return kStOK;
-
-      /*
-      //TOF mult
-      StTriggerData* trg = event->triggerData();
-      int tofMult = trg->tofMultiplicity();
-      //                cout<<"TOF mult="<<tofMult<<endl;
-      if (tofMult > 20) return kStOK;
-      */
+      
+      //TOF mult cut
       int tofMult = 0;
+      const StTriggerData* trgdata = event->triggerData();
+      if(!trgdata && StMuDst::event()) trgdata = StMuDst::event()->triggerData();
```

Comment:
So `StMuEvent` provides `StMuTriggerIdCollection`, but not `StMuTriggerData` (stores a `StTriggerData` instead).